### PR TITLE
Missed one reference to old forum URL

### PR DESCRIPTION
**Pull request by randyzwitch**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -3,5 +3,5 @@ name  : Ask a Question
 about : The issue tracker is not for questions -- please ask questions at https://community.rstudio.com/c/shiny.
 ---
 
-The issue tracker is not for questions. If you have a question, please feel free to ask it on our community site, at https://community.rstudio.com/c/shiny.
+The issue tracker is not for questions. If you have a question, please feel free to ask it on our community site, at https://forum.posit.co/c/shiny.
```
Comment (cpsievert) [inline .github/ISSUE_TEMPLATE/question.md:6]: The link with `/c/` doesn't seem to exist?

```suggestion
The issue tracker is not for questions. If you have a question, please feel free to ask it on our community site, at https://forum.posit.co/tags/shiny.
```

File: .github/ISSUE_TEMPLATE/question.md [hunk 2]
```diff
@@ -3,5 +3,5 @@ name  : Ask a Question
 about : The issue tracker is not for questions -- please ask questions at https://community.rstudio.com/c/shiny.
```
Comment (cpsievert) [inline .github/ISSUE_TEMPLATE/question.md:3]:
```suggestion
about : The issue tracker is not for questions -- please ask questions at https://forum.posit.co/tags/shiny.
```

File: .github/ISSUE_TEMPLATE/question.md (hunk 1 above)
Comment (randyzwitch) [reply .github/ISSUE_TEMPLATE/question.md:6]: Does for me...the number reference at the end isn't required

Screenshot 2024-05-01 at 11 36 55 AM

Comment (cpsievert) [reply .github/ISSUE_TEMPLATE/question.md:6]: Ahh, whoops. I think I included the `.` at the end when testing it

**Review by gadenbuie — APPROVED** [review]
Thanks!

**Comment by randyzwitch** [comment]
Can this be merged?

**Pull request merged**